pkg/types/types.go

```diff
@@ -0,0 +1,377 @@
+package types
+
+import (
+	"bytes"
+	"cmp"
+	"encoding/csv"
+	"encoding/json"
+	"fmt"
+	"path/filepath"
+	"reflect"
+	"slices"
+	"strings"
+
+	"github.com/gocarina/gocsv"
+
+	corev1 "k8s.io/api/core/v1"
+	"sigs.k8s.io/yaml"
+
+	"github.com/openshift-kni/commatrix/pkg/consts"
+	"github.com/openshift-kni/commatrix/pkg/utils"
+)
+
+type Env int
+
+const (
+	Baremetal Env = iota
+	Cloud
+)
+
+type Deployment int
+
+const (
+	SNO Deployment = iota
+	MNO
+)
+
+type Format int
```

Comment:
Is this type and following consts necessary? Seems like we only use the string format

Reply:
we are using the strings and this we are not using i think this we can remove

Reply:
yep